### PR TITLE
chore(release): publish 16

### PR DESCRIPTION
**Pull request by michaelfig**
Refs: #5049

 - @agoric/cosmos@0.29.0
 - @agoric/ertp@0.14.0
 - @agoric/swingset-vat@0.26.0
 - @agoric/access-token@0.4.18
 - agoric@0.15.0
 - @agoric/assert@0.4.0
 - @agoric/cosmic-swingset@0.35.0
 - @agoric/deploy-script-support@0.7.0
 - @agoric/deployment@2.2.1
 - @agoric/eslint-config@0.3.24
 - @agoric/governance@0.5.0
 - @agoric/import-manager@0.3.0
 - @agoric/notifier@0.4.0
 - @agoric/pegasus@0.7.0
 - @agoric/run-protocol@0.9.0
 - @agoric/same-structure@0.2.0
 - @agoric/sharing-service@0.2.0
 - @agoric/solo@0.6.0
 - @agoric/sparse-ints@0.2.0
 - @agoric/spawner@0.5.0
 - @agoric/stat-logger@0.4.26
 - @agoric/store@0.7.0
 - @agoric/swing-store@0.6.6
 - @agoric/swingset-runner@0.19.0
 - @agoric/telemetry@0.2.0
 - @agoric/ui-components@0.2.31
 - @agoric/vat-data@0.2.0
 - @agoric/vats@0.8.0
 - @agoric/wallet-connection@0.1.6
 - @agoric/wallet@0.12.0
 - @agoric/web-components@0.1.5
 - @agoric/xsnap@0.12.0
 - @agoric/zoe@0.22.0
 - @agoric/wallet-backend@0.11.0
 - @agoric/wallet-ui@0.1.0

### CODE REVIEW

**Review by dckc — APPROVED** [review]
looks ready for RUN protocol preview!

looks entirely mechanical, following https://github.com/Agoric/agoric-sdk/blob/master/MAINTAINERS.md

bummer the top level CHANGELOG is no longer updated.

File: packages/run-protocol/CHANGELOG.md
Comment (turadg) [inline packages/run-protocol/CHANGELOG.md:63]: this was unreverted (only reverted to unblock CI). It's confusing to have in the changelog and I think we should strike it.

**Pull request merged**
